ACPs/77-reinventing-subnets/README.md

```diff
@@ -127,18 +149,27 @@ type EvictSubnetValidatorTx struct {
 The `Message` field in the above transaction must be an Avalanche Warp Message using the `AddressedCall` format with the below payload. A `typeID` for distinguishing between the different payload types on the P-Chain must be included prior to this ACP being considered "Implementable".
 
 ```text
-+-----------+----------+----------+
-|      txID : [32]byte | 32 bytes |
-+-----------+----------+----------+
-                       | 32 bytes |
-                       +----------+
++----------+----------+----------+
+| subnetID : [32]byte | 32 bytes |
++----------+----------+----------+
+|   nodeID : [32]byte | 32 bytes |
++----------+----------+----------+
+|   weight :   uint64 |  8 bytes |
++----------+----------+----------+
+|    nonce :   uint64 |  8 bytes |
++----------+----------+----------+
+                      | 80 bytes |
+                      +----------+
 ```
 
+- `subnetID`, `nodeID`, and `weight` are for the Subnet Validator being modified
+- `nonce` is a strictly increasing number that denotes the latest validator weight update and provides replay protection for this transaction. The P-Chain state will store a `minNonce` associated with the warp `messageID` used in `RegisterSubnetValidatorTx` above. When accepting the `RegisterSubnetValidatorTx`, the `minNonce` was set to `0`. `nonce` must satisfy `nonce >= minNonce` for `SetSubnetValidatorWeightTx` to be valid. Note that `nonce` is not required to be incremented by `1` with each successive validator weight update. If `minNonce` is `MaxUint64`, the `weight` in the `SetSubnetValidatorWeightTx` is required to be `0` to prevent Subnets from being unable to remove `nodeID` in a future transaction. When a Subnet Validator is removed from the active set, the `minNonce` and `messageID` will be removed from the P-Chain state.
```

Comment:
We should explicitly comment that the `minNonce` will be set to `nonce + 1` during the execution of this transaction.